### PR TITLE
Update SauceLabs.mdx

### PR DESCRIPTION
**Pull request by lysannep**
Updates for readability, also fixes spacing issues.

### CODE REVIEW

File: docs/plugins/SauceLabs.mdx
Comment (VolhaKarenko) [inline docs/plugins/SauceLabs.mdx:34]: Click 'Add integration'.

**Review by VolhaKarenko — CHANGES_REQUESTED** [review]
Everything is correct, just changed the name of the button

File: docs/plugins/SauceLabs.mdx
Comment (pressayuliya) [reply docs/plugins/SauceLabs.mdx:34]: Approved from my side.

**Comment by pressayuliya** [comment]
Approved from my side.

File: docs/plugins/SauceLabs.mdx
Comment (lysannep) [reply docs/plugins/SauceLabs.mdx]: Done.

**Comment by lysannep** [comment]
Thank you @pressayuliya @VolhaKarenko ~ I added the requested change.

**Comment by alena-babrova** [comment]
Ok from my side. Approved.

**Comment by pressayuliya** [comment]
Approved from my side. Thank you.

**Comment by lysannep** [comment]
> Everything is correct, just changed the name of the button

@VolhaKarenko ~ Can you please approve the PR? I can't merge the changes until I get an official approval.

**Review by VolhaKarenko — APPROVED** [review]
Approved

**Comment by lysannep** [comment]
@VolhaKarenko Thank you! Can you also merge the PR? I'm not authorized to merge.

**Review by alena-babrova — APPROVED** [review]

**Review by pressayuliya — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**
